Sources/Extensions/UIBezierPathExtension.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  Created by phimage on 18/05/2017.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+extension UIBezierPath {
+
+  // MARK: - Circle
+
+  /**
+   Create a Bezier path for a circle shape.
+   
+   - Parameter bounds: The bounds of shape.
+   */
+  convenience init(circleIn bounds: CGRect) {
+    let diameter = ceil(min(bounds.width, bounds.height))
```

Comment:
do we need to call `self.init()` here?

Reply:
`self.init(` is called thread lines under

Reply:
@phimage cool 👍